rtengine/helpersse2.h

```diff
@@ -416,6 +424,11 @@ static INLINE vint2 vsrli2(vint2 x, int c)
 {
     return vsrli(x, c);
 }
+#else
+#define vslli2(x,c) (vint2)_mm_slli_epi32((vint2)x, (int)c)
+#define vsrli2(x,c) (vint2)_mm_srli_epi32((vint2)x, (int)c)
```

Comment:
Couldn't hurt to throw some parentheses around the parameters.
```suggestion
#define vslli2(x,c) (vint2)_mm_slli_epi32((vint2)(x), (int)(c))
#define vsrli2(x,c) (vint2)_mm_srli_epi32((vint2)(x), (int)(c))
```